spring-boot-project/spring-boot-docs/src/docs/asciidoc/howto/webserver.adoc

```diff
@@ -443,7 +443,7 @@ You can register it as a servlet filter in your application by setting `server.f
 TIP: If you are using Tomcat and terminating SSL at the proxy, configprop:server.tomcat.redirect-context-root[] should be set to `false`.
 This allows the `X-Forwarded-Proto` header to be honored before any redirects are performed.
 
-NOTE: If your application runs in Cloud Foundry or Heroku, the configprop:server.forward-headers-strategy[] property defaults to `NATIVE`.
+NOTE: If your application runs in Cloud Foundry, Heroku or K8s, the configprop:server.forward-headers-strategy[] property defaults to `NATIVE`.
```

Comment:
Thanks for the suggestion. I think it would be better to use Kubernetes here rather than the K8S abbreviation.